components/gallery/GalleryItem.vue

```diff
@@ -176,6 +176,10 @@ useNuxt2Meta({
   font-size: 2.4375em;
 }
 
+.name-container {
+  max-width: 75%;
+}
+
```

Comment:
Use linter

Reply:
prettier not picking anything wrong though

Reply:
Hey, what does `name-container` fix? I've seen that is working well without

Reply:
when its a long name, buttons starts to show up one below the other, max width fixes that

Reply:
oki